ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -171,7 +176,9 @@ export const NetworkListMenu = ({ onClose }) => {
     return sortedNonTestNetworks;
   };
 
+  // check if not granted chain is clicked, then show the toast and grant permission
```

Comment:
[nit] a bit unclear what this comment is pointing to

Reply:
It's removed now here https://github.com/MetaMask/metamask-extension/pull/26635/commits/3868d34057e993b1cbcdf570bd5765b62e4bf5e7. I added it initially for myself as a TODO